lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -66,6 +67,12 @@ def joined_prefixes
         def prefixes
           cop_config['Prefixes'] || []
         end
+
+        def prefix_regex
+          escaped_prefixes =
+            prefixes.map { |prefix| "#{Regexp.escape(prefix)}\\b" }
+          /^#{escaped_prefixes.join('|')}/
```

Comment:
In theory there's `Regexp.union`, but with `escape`, `/\b/` and `/^/` needed to be added it seems the current approach is unbeatable.

Reply:
Resolved using @bquorning's suggestion `/^#{Regexp.union(prefixes)}\b/`